README.md

```diff
@@ -103,7 +122,7 @@ npm run build
 
 ## Scripts
 
-The following npm scripts are available. With the exception of `npm start` and `npm test`, the remaining scripts can be run from your command line with `npm run scriptName`.
```

Comment:
Broken window: you can invoke `start` and `test` using `npm run` syntax, which is what we do everywhere else.